apirest.md

```diff
@@ -1220,6 +1224,197 @@ $ curl -X DELETE \
+}
+
+$ curl -X GET \
+-H 'Content-Type: application/json' \
+-H "Session-Token: 83af7e620c83a50a18d3eac2f6ed05a3ca0bea62" \
+-H "App-Token: f7g3csp8mgatg5ebc5elnazakw20i9fyev1qopya7" \
+'http://path/to/glpi/apirest.php/getMassiveActions/Computer/4'
+
+< 200 OK
+{
+  "MassiveAction:purge_item_but_devices": "Delete permanently but keep devices",
+  "MassiveAction:purge": "Delete permanently and remove devices",
+  "MassiveAction:restore": "Restore",
+  "Lock:unlock": "Unlock components"
+}
+```
+
+## Get massive action parameters
+
+* **URL**: apirest.php/getMassiveActionParameters/:itemtype/
+* **Description**: Show the availables parameters for a given massive action.
+* **Method**: GET
+* **Parameters**: (Headers)
+  * *Session-Token*: session var provided by [initSession](#init-session) endpoint. Mandatory.
+  * *App-Token*: authorization string provided by the GLPI API configuration. Optional.
+* **Parameters**: (query string)
+* **Returns**:
+  * 200 (OK).
+  * 400 (Bad Request) with a message indicating an error in input parameter.
+  * 401 (UNAUTHORIZED).
+
+Example usage (CURL):
+
+```bash
+$ curl -X GET \
+-H 'Content-Type: application/json' \
+-H "Session-Token: 83af7e620c83a50a18d3eac2f6ed05a3ca0bea62" \
+-H "App-Token: f7g3csp8mgatg5ebc5elnazakw20i9fyev1qopya7" \
+'http://path/to/glpi/apirest.php/getMassiveActionParameters/Computer/MassiveAction:add_note'
+
+< 200 OK
+{"add_note":"text"}
+
+$ curl -X GET \
+-H 'Content-Type: application/json' \
+-H "Session-Token: 83af7e620c83a50a18d3eac2f6ed05a3ca0bea62" \
+-H "App-Token: f7g3csp8mgatg5ebc5elnazakw20i9fyev1qopya7" \
+'http://path/to/glpi/apirest.php/getMassiveActionParameters/Computer/Contract_Item:add'
+
+< 200 OK
+{"peer_contracts_id":"dropdown"}
+```
+
+## Apply massive action
+
+* **URL**: apirest.php/applyMassiveAction/:itemtype/
+* **Description**: Run the given massive action
+* **Method**: GET
+* **Parameters**: (Headers)
+  * *Session-Token*: session var provided by [initSession](#init-session) endpoint. Mandatory.
+  * *App-Token*: authorization string provided by the GLPI API configuration. Optional.
+* **Parameters**: (query string)
+  * *ids* items to execute the action on. Mandatory.
+  * *specific action parameters* some actions require specific parameters to run. You can check them through the 'getMassiveActionParameters' endpoint.
+* **Returns**:
+  * 200 (OK) All items were processed.
+  * 207 (Multi-Status) Not all items were successfully processed.
+  * 400 (Bad Request) with a message indicating an error in input parameter.
+  * 401 (UNAUTHORIZED).
+  * 422 (Unprocessable Entity) All items failed to be processed.
+
+Example usage (CURL):
+
+```bash
+$ curl -X GET \
+-H 'Content-Type: application/json' \
+-H "Session-Token: 83af7e620c83a50a18d3eac2f6ed05a3ca0bea62" \
+-H "App-Token: f7g3csp8mgatg5ebc5elnazakw20i9fyev1qopya7" \
+'http://path/to/glpi/apirest.php/applyMassiveAction/Computer/MassiveAction:amend_comment?ids[0]=1&ids[1]=3&amendment=newtext'
```

Comment:
IMHO, as this is a writing method, it should be done using POST method and parameters should be sent into request body.

Reply:
Done in e67be62.
I've also improved the unit test for this endpoint to make sure the data was modified as expected.